src/scaffoldmaker/meshtypes/meshtype_3d_bladderurethra1.py

```diff
@@ -1163,6 +1164,25 @@ def generateBaseMesh(cls, region, options):
             bladderNodesetGroup.addNode(markerPoint)
             nodeIdentifier += 1
 
+        ################
+        # point markers
+        ################
+
+        markerTermNameBrainstemCoordinatesMap = {
+            # "pudendal nerve": [0.0, 1.0, 0.0],
+            # "pelvic splanchnic nerve": [0.0, 1.0, 0.0],
+            # "hypogastric nerve": [0.0, 1.0, 0.0],
+            # "lumbar splanchnic nerve": [0.0, 1.0, 0.0],
+            "fundus of urinary bladder": [14.6, 0.01, 23.1],
+            "external urethral sphincter": [-1.5, 6.7, 110.5],
+        }
+        bladder_coordinates = findOrCreateFieldCoordinates(fm)
+        nodeIdentifier = max(1, getMaximumNodeIdentifier(nodes) + 1)
+        for termName, brainstemCoordinatesValues in markerTermNameBrainstemCoordinatesMap.items():
+            annotationGroup = findOrCreateAnnotationGroupForTerm(annotationGroups, region, get_bladder_term(termName))
+            annotationGroup.createMarkerNode(nodeIdentifier, bladder_coordinates, brainstemCoordinatesValues)
+            nodeIdentifier += 1
+
```

Comment:
Firstly, this would be OK if bladder_coordinates exist, but they do not.
(And it would need to use "bladder" rather than "brainstem" so the latter word should not appear anyway.)
Currently this will only work for human and when no-one changes any central path or size parameters, since coordinates are being used as material coordinates.
We need to discuss what's best here; I think it's better to use only element & xi until actual material coordinates exist. The above code won't work for other species - the locations will be different/wrong for other species.